### PR TITLE
[16.0] [MIG] partner statement

### PR DESCRIPTION
**Pull request by odooNextev**

### CODE REVIEW

File: partner_statement/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+Users willing to access to this report should have proper Accounting & Finance rights:
+
+#. Go to *Settings / Users* and edit your user to add the corresponding access rights as follows.
+#. In *Application / Accounting & Finance*, select *Billing* or *Billing Manager*
```
Comment (mymage) [inline partner_statement/readme/CONFIGURE.rst]:
```suggestion
#. In *Access Rights/ Accounting / Invoicing*, select *Billing* or *Billing Administrator*
```

**Review by mymage — APPROVED** [review]
Functional ok.

**Comment by JordiBForgeFlow** [comment]
The module has already been merged

**Pull request closed without merging**